packages/tslint-config/README.md

```diff
@@ -0,0 +1,86 @@
+<img height="204" src="https://cloud.githubusercontent.com/assets/464822/20228152/d3f36dc2-a804-11e6-80ff-51ada2d13ea7.png">
+
+# [Blueprint](http://blueprintjs.com/) [TSLint](https://palantir.github.io/tslint) configuration
+
+Blueprint is a React UI toolkit for the web.
+
+This package contains configuration for [TSLint](https://palantir.github.io/tslint) (the TypeScript linter) and a handful of new rules specifically for use when developing against Blueprint libraries
+
+## Installation
+
+```
+yarn add @blueprintjs/tslint-config tslint
+```
+
+## Basic usage
+
+Simply extend this package in your `tslint.json` to use the default rules configuration. This configuration is applicable to any codebase (you do not have to use Blueprint packages).
+
+`tslint.json`
+```json
+{
+  "extends": "@blueprintjs/tslint-config"
+}
+```
+
+## Advanced usage
+
+To enable the Blueprint-specific rules which enforce semantics particular to `@blueprintjs` packages, add another `extends` entry.
+
+`tslint.json`
+```json
+{
+  "extends": [
+    "@blueprintjs/tslint-config",
+    "@blueprintjs/tslint-config/blueprint-rules"
```

Comment:
I would add an example with only this line and make it the basic usage example. There will be a lot more people I think who will want to just use the BP rules than use all the rules.

Reply:
i'll def add a note that the rules can be used on their own